gen/build_deploy/bash.py

```diff
@@ -654,12 +655,18 @@ def fill_template(base_name, format_args):
 def make_dcos_launch():
     # NOTE: this needs to be kept in sync with build_dcos_launch.sh
     work_dir = py.path.local.mkdtemp()
+    # the tree will not be in the working dir if run from downstream
+    tree_src = './'
+    if os.path.exists('ext/upstream'):
+        tree_src = './ext/upstream'
```

Comment:
nit: This works for now, kinda ugly but functional. Relies on magic knowledge between how the two repos code works...